sdk/core/azure-core-http-httpurlconnection/src/main/java/com/azure/core/http/httpurlconnection/HttpUrlConnectionResponse.java

```diff
@@ -0,0 +1,69 @@
+package com.azure.core.http.httpurlconnection;
+
+import com.azure.core.http.HttpHeader;
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpRequest;
+import com.azure.core.http.HttpResponse;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+
+public class HttpUrlConnectionResponse extends HttpResponse {
```

Comment:
This is not intended to be public API. Move custom response type into a sub-package named `implementation`, similar to the other http clients, or make the class 'package-private' so that it is not visible to classes outside this package.